edb/server/protocol/auth_ext/http.py

```diff
@@ -83,72 +97,67 @@ async def handle_request(
-                    return await self.handle_magic_link_email(*handler_args)
+                    await self.handle_magic_link_email(request, response)
                 case ('magic-link', 'authenticate'):
-                    return await self.handle_magic_link_authenticate(
-                        *handler_args
-                    )
+                    await self.handle_magic_link_authenticate(request, response)
 
                 # WebAuthn routes
                 case ('webauthn', 'register'):
-                    return await self.handle_webauthn_register(*handler_args)
+                    await self.handle_webauthn_register(request, response)
                 case ('webauthn', 'register', 'options'):
-                    return await self.handle_webauthn_register_options(
-                        *handler_args
+                    await self.handle_webauthn_register_options(
+                        request, response
                     )
                 case ('webauthn', 'authenticate'):
-                    return await self.handle_webauthn_authenticate(
-                        *handler_args
-                    )
+                    await self.handle_webauthn_authenticate(request, response)
                 case ('webauthn', 'authenticate', 'options'):
-                    return await self.handle_webauthn_authenticate_options(
-                        *handler_args
+                    await self.handle_webauthn_authenticate_options(
+                        request, response
                     )
 
                 # UI routes
                 case ('ui', 'signin'):
-                    return await self.handle_ui_signin(*handler_args)
+                    await self.handle_ui_signin(request, response)
                 case ('ui', 'signup'):
-                    return await self.handle_ui_signup(*handler_args)
+                    await self.handle_ui_signup(request, response)
                 case ('ui', 'forgot-password'):
-                    return await self.handle_ui_forgot_password(*handler_args)
+                    await self.handle_ui_forgot_password(request, response)
                 case ('ui', 'reset-password'):
-                    return await self.handle_ui_reset_password(*handler_args)
+                    await self.handle_ui_reset_password(request, response)
                 case ("ui", "verify"):
-                    return await self.handle_ui_verify(*handler_args)
+                    await self.handle_ui_verify(request, response)
                 case ("ui", "resend-verification"):
-                    return await self.handle_ui_resend_verification(
-                        *handler_args
-                    )
+                    await self.handle_ui_resend_verification(request, response)
                 case ("ui", "magic-link-sent"):
-                    return await self.handle_ui_magic_link_sent(*handler_args)
+                    return await self.handle_ui_magic_link_sent(
```

Comment:
I'm not sure why we dropped the `return` in the other cases, but should we do it here too?

Reply:
Good catch!